modules/gdnative/doc_classes/GDNativeLibrary.xml

```diff
@@ -39,7 +39,7 @@
 			[b]Note:[/b] If the library defines tool scripts that run inside the editor, [code]reloadable[/code] must be [code]false[/code]. Otherwise, the editor will attempt to unload the tool scripts while they're in use and crash.
 		</member>
 		<member name="singleton" type="bool" setter="set_singleton" getter="is_singleton" default="false">
-			If [code]true[/code], Godot loads the library at startup rather than the first time a script uses the library, calling [code]gdnative_singleton[/code] after initializing the library. The library remains loaded as long as Godot is running.
+			If [code]true[/code], Godot loads the library at startup rather than the first time a script uses the library, calling [code]{prefix}_gdnative_singleton[/code] after initializing the library (where [code]{prefix}[/code] is the value of [member symbol_prefix]). The library remains loaded as long as Godot is running.
```

Comment:
A small quirk of GDNative is that the underscore is considered part of the prefix. Hopefully the default value ("godot_") makes it obvious, but you might consider removing it after the `{prefix}` here.